src/index.js

```diff
@@ -1,17 +1,13 @@
-import React from 'react';
-import ReactDOM from 'react-dom';
+
+
 import './index.css';
 import App from './App';
 import reportWebVitals from './reportWebVitals';
+import { BrowserRouter } from 'react-router-dom';
+import ReactDOM from 'react-dom';
 
-ReactDOM.render(
-  <React.StrictMode>
-    <App />
-  </React.StrictMode>,
-  document.getElementById('root')
-);
 
-// If you want to start measuring performance in your app, pass a function
-// to log results (for example: reportWebVitals(console.log))
-// or send to an analytics endpoint. Learn more: https://bit.ly/CRA-vitals
-reportWebVitals();
+<BrowserRouter>
+  <App />
+</BrowserRouter>,
+document.getElementById('root')
```

Comment:
You have removed `ReactDOM.render `for some reason, so your app crashed when i opened it -> make sure you check your code before you push it to github 

``` 
ReactDOM.render(
<BrowserRouter>
  <App />
</BrowserRouter>,
document.getElementById('root')
)
```